app/src/features/projects/components/ProjectLocationForm.tsx

```diff
@@ -290,10 +293,10 @@ const ProjectLocationForm: React.FC<IProjectLocationFormProps> = (props) => {
 
         <Box className="feature-box">
           <MapFeatureList
-            features={values.location.geometry}
             mask={[mask, setMask]}
             maskState={[maskState, setMaskState]}
             activeFeatureState={[activeFeature, setActiveFeature]}
+            formikProps={formikProps}
```

Comment:
Happy for now, but I think the functions about masks and states should definitely have another clean as tech debt. I think this can be handled alot cleaner. These states are still only being made within the `ProjectLocationForm.tsx` 
So the Mapcontainer is currently not an abstract component and not resusable